test/com/puppetlabs/puppetdb/test/config.clj

```diff
@@ -210,13 +210,13 @@
 (deftest sslv3-warn-test
   (testing "output to log"
     (tu-log/with-log-output log-output
-      (let [bad-config {:jetty {:ssl-protocols "SSLv3, TLSv1, TLSv1.1, TLSv1.2"}}]
+      (let [bad-config {:jetty {:ssl-protocols ["SSLv3" "TLSv1" "TLSv1.1" "TLSv1.2"]}}]
```

Comment:
@cprice404 @camlow325 Looks like [TK-133](https://tickets.puppetlabs.com/browse/TK-133) is now more important. We're wanting to upgrade TK so that we can get [max-threads](https://tickets.puppetlabs.com/browse/TK-130) but since this is wanted in stable which is a y release, we can't break [ssl-protocols](https://docs.puppetlabs.com/puppetdb/2.2/configure.html#ssl-protocols) like we did on master.